### PR TITLE
Extra properties for Basic Containers

### PR DESCRIPTION
**Pull request by ajs6f**
For discussion before merging, due to my sadtastic PHP unskills.

### CODE REVIEW

**Comment by acoburn** [comment]
@ajs6f this is great. The reason travis is failing is because it checks that the code follows the [PSR-2](http://www.php-fig.org/psr/psr-2/) style. If you run `php vendor/bin/phpcs --standard=PSR2 src test` on your branch, you'll see the errors.

I am pretty sure that Composer has a mechanism built in such that one could just run `composer test` and accomplish the same thing. In fact, I'll add that now, myself.

**Comment by acoburn** [comment]
@ajs6f if you rebase your PR on master now and run `composer test` (or `php composer.phar test`), it will check for PSR-2 style errors and run the unit tests.

**Comment by ajs6f** [comment]
Will do tomorrow, thanks @acoburn!

**Comment by whikloj** [comment]
Also if you run `php vendor/bin/phpcbf --standard=PSR2 src test` it will automatically fix any PSR2 errors that it can. Then you're just left with the ones you need to manually correct.

**Comment by ajs6f** [comment]
@acoburn @whikloj I had just started tinkering with `phpcbf`. I'm going to add it to the `composer.json`, if that's okay with you guys.

**Comment by ajs6f** [comment]
Oh, it was already there, cool.
Okay, that last commit should correct my formatting janks and we can then fix my substantive failures.

File: src/Model/BasicContainer.php [hunk 1]
```diff
@@ -47,12 +47,22 @@ public function respond(Application $app, Request $request, array $options = arr
...
                 if ($fileInfo->isDot()) {
                     continue;
                 }
-                $filename = rtrim($subject, '/') . '/' . ltrim($fileInfo->getFilename(), '/');
-                $graph->addResource($subject, $predicate, $filename);
+                $name = $fileInfo->getBasename("." . $fileInfo->getExtension());
+                # load extra properties for this container if available or add a containment triple
+                if ($name == $extraPropsFilename) {
+                       $rdf = $fileInfo->openFile()->fread($fileInfo->getSize());
+                       //error_log("Found rdf: " . $rdf);
```
Comment (acoburn) [inline src/Model/BasicContainer.php]: This line can be removed.

Comment (ajs6f) [inline src/Model/BasicContainer.php]: Done.

File: src/app.php [hunk 2]
```diff
@@ -40,8 +40,11 @@
             $message = "Method Not Allowed";
             $headers["Link"] = TrellisConstants::READ_ONLY_RESOURCE_LINK;
             break;
+        case (500):
+                $message = "Something went wrong!\n" . $e->getTraceAsString();
```
Comment (acoburn) [inline src/app.php]: (minor) the indentation is inconsistent here, compared to the lines above.

File: src/app.php [hunk 3]
```diff
@@ -40,8 +40,11 @@
...
+            break;
         default:
-            $message = "Something went wrong";
+                $message = "Something went wrong!\n";
```
Comment (acoburn) [inline src/app.php]: (minor) indentation

File: src/app.php (hunk 2 above)
Comment (ajs6f) [inline src/app.php]: Fixed.

File: src/app.php (hunk 3 above)
Comment (ajs6f) [inline src/app.php]: Fixed.

**Comment by acoburn** [comment]
This looks good to me. I'll let it sit a bit in case @whikloj has anything more to add, but I'll plan to merge this today.

**Pull request merged**